### PR TITLE
Angle-bracket-ize tutorial/autocomplete-component.md

### PR DESCRIPTION
**Pull request by zachgarwood**
Updated the built-in components to use angle bracket syntax. See #692

### CODE REVIEW

File: guides/release/tutorial/autocomplete-component.md [hunk 1]
```diff
@@ -57,19 +54,19 @@ as |filteredResults|
...
   placeholder="Filter By City"
-}}
+>
 {{yield this.results}}
 ```
 
-The template contains an [`{{input}}`](../../templates/input-helpers/) helper that renders as a text field, in which the user can type a pattern to filter the list of cities used in a search.
-The `value` property of the `input` will be kept in sync with the `value` property in the component.
+The template contains an [`<Input>`](../../templates/input-helpers/) helper that renders as a text field, in which the user can type a pattern to filter the list of cities used in a search.
+The `value` property of the `Input` will be kept in sync with the `value` property in the component.
```
Comment (chancancode) [inline guides/release/tutorial/autocomplete-component.md]: `@value` argument of the `<Input />` component will...

File: guides/release/tutorial/autocomplete-component.md [hunk 2]
```diff
@@ -57,19 +54,19 @@ as |filteredResults|
...
 
-Another way to say this is that the `value` property of `input` is [**bound**](../../object-model/bindings/) to the `value` property of the component.
+Another way to say this is that the `value` property of `Input` is [**bound**](../../object-model/bindings/) to the `value` property of the component.
```
Comment (chancancode) [inline guides/release/tutorial/autocomplete-component.md]: ditto

**Comment by zachgarwood** [comment]
I followed @chancancode 's suggestions and did some minor reformatting.

File: guides/release/tutorial/autocomplete-component.md [hunk 3]
```diff
@@ -57,21 +54,25 @@ as |filteredResults|
...
-{{input
-  value=this.value
-  key-up=(action "handleFilterEntry")
+<Input
+  @value=this.value
+  @key-up=(action "handleFilterEntry")
   class="light"
   placeholder="Filter By City"
-}}
+>
```
Comment (chancancode) [inline guides/release/tutorial/autocomplete-component.md]: This one needs a `/>`

Comment (locks) [inline guides/release/tutorial/autocomplete-component.md]:
```suggestion
/>
```

**Comment by zachgarwood** [comment]
I implemented @chancancode / @locks 's suggested change.

**Comment by locks** [comment]
Thanks for the help!

**Pull request merged**